app/views/MainTempleteNew.scala.html

```diff
@@ -137,21 +137,23 @@
 
 @beforeContent = {
 
-
```

Comment:
just as personal preference, if the whole purpose of a boolean is to do !boolean, i'd rather change the name so the ! isn't necessary:
if(showPtaAccountNav)

Reply:
also this would align with showGds and other stuff that is already present

Reply:
agree 100% i changed it (showPtaAccountNav  :)